webpack/assets/javascripts/react_app/API.js

```diff
@@ -1,37 +1,55 @@
-import $ from 'jquery';
+import axios from 'axios';
+
+const getcsrfToken = () => {
+  const token = document.querySelector('meta[name="csrf-token"]');
+
+  if (token) {
+    return token.content;
+  }
+  // fail gracefully when no token is found
+  return '';
+};
+const onSuccess = response => response.data;
```

Comment:
Doesn't it mean we are losing all other response fields?
Why consumers can't just use 
```js
get(url).then(({data}) => ... );
```

Reply:
No it doesn't, data is the whole respond's json body, the only fields we're losing here are headers and axios config, which I don't see any use of them.

Reply:
Some API's deciding to return data in headers, mostly when it comes to access tokens.